static/Gruntfile.js

```diff
@@ -12,18 +12,17 @@ module.exports = function (grunt) {
     watch: {
       all: {
         files: ['**', '!**/node_modules/**', `!${outputDir}/**`],
-        tasks: ['build',],
+        tasks: ['build-site'],
         options: {
           spawn: false,
         },
       },
     },
   });
-
   grunt.loadNpmTasks('grunt-webpack');
```

Comment:
ok THIS time we can get rid of grunt-webpack right?

Reply:
no, we still need to be able to call `grunt webpack` since the CI scripts call it

Reply:
can't the ci scripts call the webpack cli?

Reply:
the ci scripts should probably just do an `npm run build`

Reply:
If we remove it then any site would break if they upgraded to this version of the theme. We would need to create some patch files for the CI scripts. The CI scripts only call `grunt webpack` in one-time scenarios such as the build step, so the memory leak isn't is problem there

Reply:
I think I would like to not keep around old dependencies we don't need if possible, but we can kick the can down the road a little!

Reply:
We can definitely ask Hitchhikers to manually update their ci scripts at some point in time when they upgrade, but you're right probably not worth it here

Reply:
Maybe it's worth considering having the CI files part of theme 2.0?